live-examples/css-examples/basic-box-model/clear.css

```diff
@@ -0,0 +1,21 @@
+.example-container {
+    background-color: #eee;
+    border: .75em solid;
+    padding: .75em;
+    text-align: left;
+}
+
+.floated-left {
+    border: solid 10px #FFC129;
```

Comment:
We should have lowercase for hex values, here and elsewhere in this PR.

Reply:
fixed.